### PR TITLE
Support Swift Package Manager

### PR DESCRIPTION
**Pull request by hashknot**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// swift-tools-version:4.0
+import PackageDescription
+
+let package = Package(
+    name: "ImgixSwift",
+    products: [
+        .library(name: "ImgixSwift", targets: ["ImgixSwift"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/IBM-Swift/CommonCrypto", from: "1.0.0"),
```
Comment (hashknot) [inline Package.swift:10]: Prior to merging, I was thinking of forking this repository in our org. Thoughts?

File: Sources/ImgixSwift/include/CommonCrypto/module.modulemap [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module CommonCrypto {
+  header "/usr/include/CommonCrypto/CommonCrypto.h"
```
Comment (hashknot) [inline Sources/ImgixSwift/include/CommonCrypto/module.modulemap:2]: I'm skeptical of hard coding a path here but this was the only way I was able to get it work.

File: Package.swift (hunk 1 above)
Comment (jayeb) [reply Package.swift:10]: Normally I'd be opposed to it, but after checking out the repo I think it's not a bad solution. Seems like a pretty thin wrapper, so it probably won't take too much effort to "maintain".

**Comment by nathan-heskia** [comment]
I'd recommend checking for compatibility with these two libraries since they seem more reliable:

https://github.com/soffes/CommonCrypto

https://github.com/soffes/Crypto

**Review by nathan-heskia — APPROVED** [review]

**Comment by hashknot** [comment]
A build error needs to be resolved. Closing for now.

**Pull request closed without merging**